contributing-docs/03a_contributors_quick_start_beginners.rst

```diff
@@ -56,7 +56,7 @@ For Breeze (local development):
 * 4GB RAM, 40GB disk space, and at least 2 CPU cores
 
 .. note::
-   Docker **or Podman** installation varies by OS. See the `full guide <03b_contributors_quick_start_seasoned_developers.html#local-machine-development>`_ for Ubuntu, macOS, and Windows instructions.
+   Docker **or Podman** installation varies by OS. See the `full guide <03b_contributors_quick_start_seasoned_developers.rst#local-machine-development>`_ for Ubuntu, macOS, and Windows instructions.
```

Comment:
I feel we probably should link to `03_contributors_quick_start` instead

Reply:
It was first introduced here: https://github.com/apache/airflow/pull/55930/files#diff-a1042dc8753272a9932e8599e84031efb7236b94e06c99770462dfc0dd16d87fR29. However, GitHub cannot include `..include` correctly. Therefore, we should consider how to improve it further.